### ISSUE TITLE
Game hangs on entering X8

### ISSUE THREAD

**Issue by unabomber14** [Swartz27/xray-16#1653]
**Describe the bug**
Game hangs on loading screen on pressing the elevator button

**To Reproduce**
1. Use my savegame 
2. Go to X8 elevator
3. Press the elevator button

**Expected behavior**
Descending into the X8 lab.

**BugTrap error report**
this is the log where I killed the process [openxray_duma.log](https://github.com/OpenXRay/xray-16/files/14898028/openxray_duma.log)
this is the log where i let the game hang [openxray_duma.log](https://github.com/OpenXRay/xray-16/files/14898100/openxray_duma.log)
user.ltx: https://disk.yandex.ru/d/29Kl4uE_eYV8vg
my savegame: https://disk.yandex.ru/d/Nu-tees5r_YY8Q

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - OpenXRay build from AUR. openxray 1.6.02_2088-1

**Comment by Obiwanmonrobi** [comment]
I'm able to enter X8 but the game crashes within a few seconds after I enter the doorway into the lab. I tried loading the saved game multiple times. I also tried loading an older save game, then going to X8 but that crashed as well.  I'm running Windows 10 Pro 22H2; OpenXRay Rx64 build 7568, Oct 26 2019, AppVeyor build 558 from commit
[OpenXRay_error_report_240407-181708.zip](https://github.com/OpenXRay/xray-16/files/14899226/OpenXRay_error_report_240407-181708.zip).  I noticed that my version is from 2019 so I'm updated to the latest stable version of OpenXRay.  My saved games do not work with the new version so I've started over from the beginning again.
